guides/micronaut-openapi-generator-server/java/README.md

```diff
@@ -0,0 +1,23 @@
+# openapi-micronaut
```

Comment:
Should we have this file as part of the complete solution? Or should we delete it?